src/blueprints/tv2/action-factories/tv2-transition-effect-action-factory.ts

```diff
@@ -50,8 +51,10 @@ export class Tv2TransitionEffectActionFactory {
     private readonly videoMixerTimelineObjectFactory: Tv2VideoMixerTimelineObjectFactory,
     private readonly videoClipTimelineObjectFactory: Tv2VideoClipTimelineObjectFactory,
     private readonly audioTimelineObjectFactory: Tv2AudioTimelineObjectFactory,
+    private readonly loggerService: Tv2LoggerService,
     private readonly assetPathHelper: Tv2AssetPathHelper
   ) {
+    loggerService.tag(Tv2TransitionEffectActionFactory.name)
```

Comment:
The tag has no effect in this case, since `.tag` creates a new context for the logger. The correct way to set it is:
```typescript
class MyClass {
  private readonly logger: Tv2Logger

  public constructor(logger: Tv2Logger) {
    this.logger = logger.tag(MyClass.name)
  }
}
```

Reply:
Tagging are now used correctly.